bundles/org.openhab.binding.fsinternetradio/src/test/java/org/openhab/binding/fsinternetradio/test/FSInternetRadioHandlerJavaTest.java

```diff
@@ -103,7 +102,8 @@
      * A HashMap which saves all the 'channel-acceppted_item_type' pairs.
      * It is set before all the tests.
      */
-    private static Map<String, String> acceptedItemTypes;
+    private static HashMap<String, String> acceptedItemTypes;
+    private static HashMap<String, String> ACCEPTED_ITEM_TYPES = acceptedItemTypes;
```

Comment:
It is cleaner and provides more flexibility if you make your fields and variables the collection interfaces instead of concrete implementations.
```suggestion
    private static Map<String, String> acceptedItemTypes;
    private static Map<String, String> ACCEPTED_ITEM_TYPES = acceptedItemTypes;
```

Reply:
changed